pkg/estimator/pb/types.go

```diff
@@ -1,7 +1,7 @@
 package pb
 
 import (
-	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
If modified, the grpc codes and proto files should be re-generated.

Reply:
@Garrybest we need a verification script to check if the proto file is updated to date.

Reply:
Sure.

Reply:
Just created #729 to track.